katsdpcontroller/generator.py

```diff
@@ -525,10 +525,10 @@ def get_baseline_index(a1, a2):
                    depends_resolve=True, depends_init=True, depends_ready=True)
         g.add_edge(xbgpu, dst_multicast, port='spead', depends_resolve=True)
         xbgpu.command += [
-            f'{{endpoints_vector[multicast.{acv.name}_spead][{i}].host}}',
-            f'{{endpoints_vector[multicast.{acv.name}_spead][{i}].port}}',
-            f'{{endpoints_vector[multicast.{stream.name}_spead][{i}].host}}',
-            f'{{endpoints_vector[multicast.{stream.name}_spead][{i}].port}}'
+            f'{{endpoints_vector[multicast.{acv.name}_spead][{i}].host}}:\
+                {{endpoints_vector[multicast.{acv.name}_spead][{i}].port}}',
+            f'{{endpoints_vector[multicast.{stream.name}_spead][{i}].host}}:\
+                {{endpoints_vector[multicast.{stream.name}_spead][{i}].port}}'
```

Comment:
`Endpoint.__str__` already knows how to format an endpoint as host:port, so you can just do
```suggestion
            f'{{endpoints_vector[multicast.{acv.name}_spead][{i}]}}',
            f'{{endpoints_vector[multicast.{stream.name}_spead][{i}]}}'
```